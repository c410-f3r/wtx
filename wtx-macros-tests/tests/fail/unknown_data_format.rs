#[wtx_macros::pkg(data_format(fdsdfs), id(Foo))]
mod pkg {
  #[pkg::req_data]
  struct Req;

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
