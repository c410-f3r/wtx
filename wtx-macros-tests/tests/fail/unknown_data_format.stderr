error: Unknown data format.
 --> tests/fail/unknown_data_format.rs:1:1
  |
1 | #[wtx_macros::pkg(data_format(fdsdfs), id(Foo))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `wtx_macros::pkg` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 1 previous error

